walkthroughs/1-ionic-showcase/walkthrough.adoc

```diff
@@ -0,0 +1,345 @@
+. Go offline 
+. Edit Task on first instance
+. Go to second instance of the application
+. Edit Task on second instance
+. At this point we should have server side updated by second instance of the application
+and first application not knowing about that change because it went offline.
+. This operation will result in conflict. 
+. Once we make first client online again - data conflict will occur.
+We should see popup and actuall server side data being replicated back to the client (conflict was resolved automatically)
+
+. Use the “Manage Tasks” page in the showcase client app.
+. Try create one task in one device, and it should show up immediately in another.
+. Try bring one client offline (, perform some actions, bring it online and you should see the changes synced to another app straight away.
+. To create conflicts, try bringing both clients offline, and modify the same item. The last instance of app that is online will receive conflicts.
+
+[time=35]
+== [Optional] Check the Data Synchonization Audit Logs
+
+You can use the OpenShift logging feature to see the audit logs that are generated by the syncserver app.
+
+=== Steps
+
+. Find out the URL of the Kibana dashboard of your OpenShift cluster:
+.. Run the following commands on the bastion server (as root user):
++
+----
+oc project openshift-logging
+oc get route
+----
++
+.. You should see there is a route called `logging-kibana`. Copy the `Host` value of that route and open it in your browser. You should see the Kibana dashboard page.
+. Filter out the audit log messages in Kibana. 
+.. On the `Discover` page, there should be a dropdown you can select on this page. Choose the namespace where the syncserver app is deployed to, and take a look at the messages.
+.. You should see a lot of messages, try expand one and see what the message looks like.
+.. In the `Search` field on the top, enter `tag:AUDIT` and you should be able to see the audit logs generated by the app. If you can't see any message, go to the GraphqQL playground page and execute some queries and then try search again.
+.. [Optional] Follow the steps descibed in link:https://github.com/aerogear/apollo-voyager-server/blob/master/doc/guides/metrics-and-audit-logs.md#importing-kibana-saved-objects:[Importing Kibana Saved Objects guide] and try importing a dashboard template for the sync app, and view the imported dashboard.
+
+
+[time=50]
+== [Optional] Protect the sync app by Keycloak
+
+The sync server app is not protected at the moment, let's bind it with Keycloak and protect the endpoints.
+
+=== Steps
+
+// TODO this is APB based. We need operator ones?
+// We need to figure out steps when keycloak operator will be ready
+
+. Go to the link:{openshift-host}/console/project/{walkthrough-namespace}/overview[OpenShift conosle, window="_blank"], find `Identity Management` in the "Provisioned Services" section, and click on "Create Binding".
```

Comment:
@grdryn Do you know the state of the indentity management operator. 
I would like to see how this changes after migration from APB to Operator and adjust this steps.

Reply:
I'm not 100% sure about that. We're not creating our own "identity management" operator like we had an APB for that...there will just be some identity management stuff already in integreatly. It might be the 'user-sso' project that you'd see on an integreatly cluster, or something else. I think @wei-lee would be the best person to help you answer that!

Reply:
yes that's what the `user-sso` project is for there, which uses the existing keycloak operator created by integreatly

Reply:
so mdc will actually create two crs for that operator when "binding" the "identity service".

this work includes a change in mdc in supporting using the keycloak operator crs instead of service binding from the ansible service catalog.

Reply:
Ah.. so it will be MDC binding that will do the magic on the. 
That is really neat. Just getting more familiar with the entire area so I want to apologize to such an obvious question. 

I will play with that on my integrately cluster to get a better feeling about it. Thank you guys!